### PR TITLE
Add template for images for invisible characters

### LINKED ISSUE #340: Images for invisible, whitespace, and combining characters
https://w3c.github.io/tr-design/src/README#unicode-codepoints

The stylesheet does not include support for images in the character template. See https://www.w3.org/TR/international-specs/#example-example-of-a-character-reference-with-an-image

### PR DESCRIPTION
**Pull request by aphillips**
Fixes #340.

- Adds the text about using an `img` for invisible/whitespace characters
- Adds the corresponding style
- Adds an image for U+00A0, used in the example
- Fix a typo for the second subsection number

### CODE REVIEW

**Comment by aphillips** [comment]
@fantasai @xfq @r12a Please check this.

File: src/base.css [hunk 1]
```diff
@@ -1371,9 +1371,15 @@ Possible extra rowspan handling
...
+	
+	.codepoint img {
+	    height: 2em;
+        margin-inline: 0.25em;
+        vertical-align: bottom;
+	}
 
 	.uname {
-		font-family: Arial, monospace;
+        font-family: Arial, monospace;
```
Comment (deniak) [inline src/base.css]: Please use tabs for indentation.

File: src/README.html [hunk 2]
```diff
@@ -981,6 +973,27 @@ <h5 id="unicode-codepoint-simple">
...
+  <p>Filling in the above template like this:</p>
+
+  <pre>
+<!-- -->&lt;span class="codepoint" translate="no">&lt;img alt="NBSP" src="images/00A0.png">
+<!-- -->&lt;code class="uname">U+00A0 NO-BREAK SPACE&lt;/code>&lt;/span>
+  </pre>
+
+  <p>Produces output in the page like this: <span class="codepoint" translate="no">
+	  <img alt="NBSP" src="images/00A0.png"><code class="uname">U+00A0 NO-BREAK SPACE</code></span>.</p>
+</aside>
```
Comment (deniak) [inline src/README.html]:
```suggestion
  <aside class="example" title="Example of a character reference with an image">
    <p>Filling in the above template like this:</p>

    <pre>
<!-- -->&lt;span class="codepoint" translate="no">&lt;img alt="NBSP" src="images/00A0.png">
<!-- -->&lt;code class="uname">U+00A0 NO-BREAK SPACE&lt;/code>&lt;/span>
    </pre>

    <p>Produces output in the page like this: <span class="codepoint" translate="no">
	  <img alt="NBSP" src="images/00A0.png"><code class="uname">U+00A0 NO-BREAK SPACE</code></span>.</p>
  </aside>
```

**Review by deniak — CHANGES_REQUESTED** [review]

File: src/base.css (hunk 1 above)
Comment (aphillips) [reply src/base.css]: Done

**Review by r12a — APPROVED** [review]

**Review by deniak — APPROVED** [review]

**Pull request merged**